desktop/cmp/persistenceManager/impl/ManageDialogModel.ts

```diff
@@ -0,0 +1,243 @@
+import {FormModel} from '@xh/hoist/cmp/form';
+import {GridAutosizeMode, GridModel} from '@xh/hoist/cmp/grid';
+import {HoistModel, managed, XH} from '@xh/hoist/core';
+import {lengthIs, required} from '@xh/hoist/data';
+import {Icon} from '@xh/hoist/icon';
+import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {includes} from 'lodash';
+import {PersistenceManagerModel} from '../PersistenceManagerModel';
+
+export class ManageDialogModel extends HoistModel {
+    parentModel: PersistenceManagerModel;
+
+    @bindable isOpen: boolean = false;
+
+    @managed readonly gridModel: GridModel;
+
+    @managed readonly formModel: FormModel;
+
+    get selectedId(): string {
+        return this.gridModel.selectedId as string;
+    }
+
+    get selIsShared(): boolean {
+        return this.gridModel.selectedRecord?.data.isShared ?? false;
+    }
+
+    get userCreated(): boolean {
+        return this.gridModel.selectedRecord?.data.createdBy === XH.getUser().username;
+    }
+
+    get canDelete(): boolean {
+        return this.parentModel.views.length > 1 && (this.canManageGlobal || !this.selIsShared);
```

Comment:
Along the lines of `"Restore Default View"`, seems like maybe we should support deleting the last view for apps that allow that.